integrationtests/spring-boot-it/pom.xml

```diff
@@ -19,6 +19,24 @@
         <overridden.version.mockito>1.10.19</overridden.version.mockito>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.infinispan</groupId>
+                <artifactId>infinispan-bom</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>1.5.16.RELEASE</version>
```

Comment:
we can keep the version and the property, but not in the bom, but in the build-configuration pom.xml, because it's the parent pom of infinispan

Reply:
@jsenko